Content: Unfettered campaign Act 1 (#6416)
endless-sky/endless-sky: data/hai/unfettered 2 war preparations.txt

				to display
					not "unfettered: Mera line"
			branch speak
				not "unfettered: Merora translate"
			`	Thankfully you spot Merora, the young Hai that helped you translate some time ago, back in the temple on this very same planet, when the elder's meeting with the public was happening.`
				to display
					has "unfettered: Merora translate"
			`	As always you have your translator with you, ready for action.`
				to display
					has "language: Hai"
			`	You also spot a translator or two near the stage, apparently for public use. They may be there for the occasional pirate the Unfettered may make use of, but there does not seem to be anyone using them here.`
				to display
					not "language: Hai"
			choice
				`	(Go speak with Merora.)`
					to display
						has "unfettered: Merora translate"
				`	(Grab the translator.)`
					to display
						not "language: Hai"
					goto speaktake
				`	(Use your translator.)`
					to display
						has "language: Hai"
					goto speak

> 			`	You notice she seems to have made a few friends since the last time you saw her, and she is so busy discussing with them that she hardly notices you until you are right in front of her. She stands there for a few seconds, maw open, before gaining back focus and smiling with joy. "<first>! It's been a while." She presents her comrades to you, "This is Rick, a human that grew up in Hai space and then proved himself through other means than battles." You shake hands and present yourself. "And this is Kenep, she left Hai space some time ago, to get inspiration for her art. They work on the same ship I do. To be frank we don't see battle often, but I've had my fair share of it already, so I'm more than happy with that!" She says the last part with a bit of shame, but honesty.`

Reviewer: ```suggestion
			`	You notice she seems to have made a few friends since the last time you saw her, and she is so busy talking with them that she hardly notices you until you are right in front of her. She stands there for a few seconds, mouth open, before gaining back focus and smiling with joy. "<first>! It's been a while." She presents her comrades to you, "This is Rick, a human that grew up in Hai space and then proved himself through other means than battles." You shake hands and present yourself. "And this is Kenep. She left Hai space some time ago, to get inspiration for her art. They work on the same ship I do. To be frank we don't see battle often, but I've had my fair share of it already, so I'm more than happy with that!" She says the last part with a bit of shame, but honesty.`
```

				to display
					has "unfettered: used translator"
			label choice
			choice
				`	"I'll go collect the minerals, then."`
					accept
				`	"How should I collect minerals from asteroids?"`
					to display
						not "unfettered: asteroid tuto"
				`	"From which systems can I get those resources from?"`
					to display
						not "unfettered: asteroid systems"
					goto where
				`	"Mining is not my thing. You'll have to do this one without me."`
					goto nobiggie
			label asteroidtuto
			action
				set "unfettered: asteroid tuto"
			`	She seems a bit surprised, "You've never mined asteroids? I recommend you install an asteroid scanner so your ship can target them automatically. Just try to keep your ship stable around one and shoot it as you would any other ship. Keep in mind they have no shields, and you might not want to waste missiles on them, since they don't shoot back. If you ever run out of targets, you can fly around for a couple of days and come back; new asteroids will likely enter the system in that time."`
				goto choice
			label where
			action
				set "unfettered: asteroid systems"

> 			`	Lead can simply be found right over our heads, and there are silver asteroids present in Ehma Ti. The lead asteroid looks similar to copper but is bigger. You can also just differentiate them with an asteroid scanner. Silver is easy to recognize with its very small size and natural sturdiness. As for titanium, I know there is some in Ik'kara'ka, up North, I can point the location on your map if you want."`

Reviewer: ```suggestion
			`	Lead can simply be found right over our heads, and there are silver asteroids present in Ehma Ti. The lead asteroid looks similar to copper but is bigger. You can also just differentiate them with an asteroid scanner. Silver is easy to recognize with its very small size and natural sturdiness. As for titanium, I know there is some in Ik'kara'ka, up north. I can point out the location on your map if you want."`
```

				goto choice
			label nobiggie

> 			`	"Oh, I see." Her tail movement stops at once. "Are you sure? This would mean a lot for me, a lot of Unfettered refuse to collect resources this way already, preferring to shoot enemies that fire back."`

Reviewer: ```suggestion
			`	"Oh, I see." Her tail movement stops at once. "Are you sure? This would mean a lot for me, because a lot of Unfettered refuse to collect resources this way, instead preferring to attack enemies that shoot back."`
```

			choice
				`	"Alright, I changed my mind."`
					goto choice

> 				`	"Yes but this does not mean I will not help the Unfettered anymore."`

Reviewer: ```suggestion
				`	"Yes, but this does not mean I will not help the Unfettered anymore."`
```

					decline
	on decline
		"unfettered rep: Mera" -= 2
	npc
		system "Ik'kara'ka"
		government "Hai (Unfettered)"
		personality harvests mining uninterested staying marked
		ship "Shield Beetle" "Domecatus"
	npc
		government "Hai (Unfettered)"
		personality mining uninterested launching
		ship "Lightning Bug" "The Ten Compatriots"
		on kill
			clear "unfettered: Merora translate"
	npc
		system "Wah Yoot"
		government "Hai (Unfettered)"
		personality harvests uninterested staying entering
		ship "Water Bug" "Hoarder"
	to fail
		has "unfettered: outcast"
	on fail
		dialog phrase "Essential Unfettered Mission Fail"
	on visit

> 		dialog "You have landed, but you do not mined enough materials from asteroids yet, or they are in one of your escorts that is not in the system. Remember that silver can be acquired in Ehma Ti, whilst lead can be found in Wah Yoot."

Reviewer: ```suggestion
		dialog "You have landed, but you have not mined enough materials from asteroids yet, or they are in one of your escorts that is not in the system. Remember that silver can be acquired in Ehma Ti, whilst lead can be found in Wah Yoot."
```

	on complete
		clear "unfettered: asteroid tuto"
		clear "unfettered: asteroid systems"
		log `Gathered materials for the Unfettered shipyards. They're building sizable ships here.`
		outfit "Lead" -50
		outfit "Silver" -20
		payment 500000
		conversation
			`You bring your cargo directly to the Unfettered shipyards, and you can see huge ships being built in the distance - it might be worth checking on another day, after they put some of the materials you brought into practice. You also notice that not all materials are brought to the shipyards, some are simply sent to more rural areas.`
			`	One of the Unfettered team leaders comes to speak to you, "My greatest thanks for these materials, they will be crucial in repairing our houses, probably more so than for our ships. Please, if you have the time, try to bring us more materials, we post missions on the job board when we need them the most. In the meantime, take these <payment> as a token of my gratitude."`

mission "<uhai> 10: Titanium"

> 	name `Gather Titanium`

Author: nah it shouldn't, not the way the mission is written at least
I'd rather make it another job

> 	name `Gather Titanium`

Reviewer: Yeah, I suppose Mera always being there doesn’t quite work…

			`	"It seems like you have some cunning after all."`
				to display
					has "unfettered: monke!"
			label noboughtship
			choice
				`	"Does this mean I can buy the new big ships now?"`
				`	"What about my pay, for the tribute?"`
					goto tribute
			`	He seems amused. "They are not [completed, functional] and for that you would have to do a bit more than bringing some intel. We still don't have much [capacity, resources] to build ships. But that will come soon - I first have to do some [managing, reordering]." He may have exaggerated how quickly you will get access to the new ships. As for the outfitter access it may take one more mission to make use of these upgrades in practice.`
			choice
				`	(Go.)`
					decline
				`	"What about my pay, for the tribute?"`
			label tribute
			branch monkey
				"unfettered: monke!" > 1
			action
				payment 400000
			`	He seems a bit surprised. "Oh, right. I had forgotten about that. Here: <payment>."`
				accept
			label monkey
			`	"You expect a pay? If you want money I am sure you can find jobs in the spaceport, but you have not proven yourself worthy of a pay."`

> 			label failed

Author: nice catch!

					goto suspect
			`	He gives you a VIP tour of the whole shipyard facility, which turns out to be too good to be true. The tour ends at the local militia office, which has some questions about that scanner they find on you...`
				decline
			label buyship
			`	"Oh, is that so? Well, normally I can't sell ships, but I could make an exception for you... I have some ships that, unlike the rest, have already endured the test of time! They come at less than half the price, too. But you do understand I cannot sell you a warship." He may mean you'll have to pay more for said ship. Better prepare your bank account! You think he would probably suspect something if you backed out now. Noticing your hesitation, he adds, "There's no issue if you don't have the money right now. I can offer you a loan of 5 million, with a small... bonus."`
			label changedmind
			choice
				`	"I would like a Centipede."`
					goto centipede
				`	"This Geocoris seems like brand new tech. I like that."`
					goto geocoris
				`	"I will take a Shield Beetle, new ships just don't beat that one!"`
					goto sb
				`	"I'd like to take that loan offer."`

> 					goto loan

Author: no but I think it's fine

	on visit
		dialog phrase "generic missing stopover or cargo"
	to fail
		has "unfettered: outcast"
	on fail
		dialog phrase "Essential Unfettered Mission Fail"
	on complete
		log `Helped the Unfettered upgrade their shipyards and received a prototype ship as thanks.`
		event "unfettered: darkcloak outfitter upgrade" 30
		conversation
			`After unloading all the cargo, you go check for He-Yi in the shipyard, but instead you find a Herald, ready to rely a message to you, "Ah, <first>. You have completed your part of the bargain, now you may have your reward.`
			branch "has beetle"
				has "<uhai> 15: Modified Beetle: done"
			`	"This will help Mera prepare your reward, should you prove yourself to be worthy of it.`
			choice
				`	(Say nothing.)`
					goto "has beetle"
				`	Have I not done enough already to prove myself worthy?`
			action
				"unfettered rep: Herald" --
			`	"Being worthy is not a permanent status of some kind that gives you access to all privileges. It is something you need to keep fighting for! Besides, do you really want to deny an engineer the opportunity to have some fun with their new prototype? I think not."`
			label "has beetle"

> 			`	"There could be more opportunities on Darkcloak for you, we do not only buy Jump Drives against money.`

Reviewer 2: Should be lower case.

				to display
					not "Unfettered Jump Drive Trading: offered"
			`	"He-Yi also wants you to know you should check the job board for more jobs like this one, and be mindful that it will take a while to improve the outfitter with these new materials and schematics you provided. We may use more of your help once that will be the case." As soon as they have delivered their message, they leave without giving you a chance to ask more questions.`
			branch accept
				not "unfettered: light on the flame"
			choice
				`	"I was trailed by some disguised pirates."`
				`	(Don't tell him.)`
					goto heknows
			action
				"unfettered rep: He-Yi" ++
				payment 200000
			`	He takes a look at the details of the ship that was following you, "Indeed, they must have found this activity [suspicious, worthy of attention], or maybe they were trailing you, looking for an opportunity to try and steal your cargo. Information is [power, potential]." He is interrupted by an engineer, before quickly addressing you again, "Here, <payment> for your trouble."`
				accept
			label heknows

> 			`	You were leaving when he says something out-loud, probably not expecting an answer, "The [fire, flame] has been extinguished, why do you [conceil, hide] it?" When you look around again, he is gone.`

Reviewer 2: ```suggestion
			`	You were leaving when he says something out-loud, probably not expecting an answer, "The [fire, flame] has been extinguished, why do you [conceal, hide] it?" When you look around again, he is gone.`
```

					has "<uhai> 13: Better Translator: done"
			`	"I really wish we could improve the translator, somehow. Maybe there is someone of importance in the council that would be interested in this?"`
				to display
					not "<uhai> 13: Better Translator: done"
			choice
				`	"That can't be hard. I'll go look for human coolers then."`
					goto quantity
				`	"Can you be more specific?"`
			`	"Not really. I don't know all kinds of cooling your species could have, but it most be large, considering your technology is not as heat efficient as ours. You're probably even using things like water, ha! Don't bring us something quite that primitive. We want the good stuff, like your nitrogen cooling, for instance. Do bring a lot of it, say, 300 tons."`
			choice
				`	"I'll go look for those, then."`
					accept
				`	"I'm not interested in helping you anymore."`
					decline
			label quantity
			`	"Be sure to bring back 300 tons worth of, say, nitrogen coolers, so we have a solid base to work on. Don't bring us things like simple, water-based systems."`
				accept
	to fail
		has "unfettered: outcast"
	on fail
		dialog phrase "Essential Unfettered Mission Fail"
	on visit

> 		dialog "You do not yet have 300 tons of nitrogen human coolers that Mera requested in your cargo hold or installed on your ships. In order to load directly into your cargo hold, you need to first deselect your ships by ctrl+clicking on them."

Author: yeah ik but gotta make sure the players know no?

> 		dialog "You do not yet have 300 tons of nitrogen human coolers that Mera requested in your cargo hold or installed on your ships. In order to load directly into your cargo hold, you need to first deselect your ships by ctrl+clicking on them."

Reviewer: Doesn’t the player get told earlier in the campaign? If not, then your way is probably better, but I think they do. 
Also, I know they do in FW, and a lot of new players aren’t going to be finding Hai space before FW, and almost none this before then.

> 		dialog "You do not yet have 300 tons of nitrogen human coolers that Mera requested in your cargo hold or installed on your ships. In order to load directly into your cargo hold, you need to first deselect your ships by ctrl+clicking on them."

Author: they do it in FW? all the same I need it too, and they do it in remnants too
I placed it in parenthesis

> 		dialog "You do not yet have 300 tons of nitrogen human coolers that Mera requested in your cargo hold or installed on your ships. In order to load directly into your cargo hold, you need to first deselect your ships by ctrl+clicking on them."

Reviewer: I think it would be kind of obvious how to do it, but if you don’t know and this tells you, that’s probably good. This can be resolved.

			`	"There is another way, but it's no fun. I could just pay that cargo, instead of paying you to get it... But the captain asked for a stupid amount of money for it. So in that case, I want a compensation of <payment> from you, matey! My new ship is not going to buy itself."`
			label bloodshed
			choice
				`	"A small amount to pay to avoid needless bloodshed."`
				`	"How about you do this one on the house, for a friend?"`
					goto friend
				`	"I'd prefer to hunt it down, in that case."`
					goto tellmore
			action
				payment -1000000
				"unfettered rep: Si-Ra" --
			`	He doesn't seem too convinced about that, but agrees on the deal. He really seems disappointed about it, and you suspect he may even pay someone else to do it instead of you.`
				decline
			label friend
			branch eh
				"unfettered rep: Si-Ra" < 1
			action
				"unfettered rep: Si-Ra" --
			`	"Yes, I guess so. But I ain't giving favors all the time, mind you!"`
				decline
			label eh
			branch hate
				"unfettered rep: Si-Ra" <= -2
			`	"Yeah sure you can tell yourself that."`
				goto bloodshed
			label hate
			branch crazy
				has "unfettered: Si-Ra crazy"
			action
				set "unfettered: Si-Ra crazy"

> 			`	"Friends?" He says with a rising tone. "You think we are FRIENDS? You would do well to stay out of my way, monkey <first>!" He says, getting unconfortably close to you before backing off.`

Reviewer 2: ```suggestion
			`	"Friends?" He says with a rising tone. "You think we are FRIENDS? You would do well to stay out of my way, monkey <first>!" He says, getting uncomfortably close to you before backing off.`
```

			branch bigfavor
				"unfettered rep: Mera" >= 3
			branch simpleprize
				"unfettered rep: Mera" == 1
			action
				outfit "Double Plasma Core"
			`	"I am sure this Double Plasma Core will be very helpful to power the ship! Be careful to not let it overheat, however, I did not equip it myself because of that limitation, but I am sure you can easily obtain the necessary coolers."`
				goto ship
			label bigfavor
			action
				outfit "Korath Inferno"
			`	"I hear the Koraths call this one the 'Inferno'! Surely this will help you upgrade a ship of your choosing. I had to pull of a few strings to be authorized to give it to you," she says with a movement of her ears.`
				goto ship
			label simpleprize
			action
				outfit "Korath Heat Shunt"
				set "unfettered: sestor gift"
			`	"The Koraths wanted you to have this. They told me they made it themselves, with our 'primitive' parts, which is an accomplishment on its own!"`
			label ship
			action
				set "unfettered: prize right away"
			`	"You can go claim the ship itself tomorrow, on Darkcloak, once I am done with the repairs. It should not take long!"`
				accept
			label hera
			action
				set "unfettered: Hera"

> 			`	She almost jumps of excitement, "I was hoping you would ask that! Listening to the Alphas, I gathered Hera is the goddess of protection. That is the best way to describe this ship: it has significantly improved shielding compared to a standard Shield Beetle, and its equipement is mostly defense focused, using ionic prototypes to scramble enemy weapons and even disable their reactor, after some time. It is also my hope that this ship will be used sparingly, avoiding casualties if possible. But let us go back to the outfit upgrades I mentioned," she says whilst looking around, distracted.`

Reviewer 2: ```suggestion
			`	She almost jumps of excitement, "I was hoping you would ask that! Listening to the Alphas, I gathered Hera is the goddess of protection. That is the best way to describe this ship: it has significantly improved shielding compared to a standard Shield Beetle, and its equipment is mostly defense focused, using ionic prototypes to scramble enemy weapons and even disable their reactor, after some time. It is also my hope that this ship will be used sparingly, avoiding casualties if possible. But let us go back to the outfit upgrades I mentioned," she says whilst looking around, distracted.`
```

				goto delaysys
			label hull
			`	"Are you intending to use it against the soulless machines to the east, or did you get intimidated that badly by the piercing weapon of earlier missions? Whatever the case, do not worry, I know how to address that, and more.`
				goto delaysys
			label shield
			`	"I thought I had already placed enough shield generators, 3 of them in fact... But you're the one who will be flying that ship so I will do as you suggest. Indeed, I think I know of a way to address that, and other weaknesses I've found when piloting the ship has as well.`
			label delaysys
			action
				event "unfettered: syscore's got it" 10
			label delay
			`	"Of course, this means that it will take a bit before you can receive the ship. I will send you a notice when that is the case, I do not expect the preparations to last more than a week or two. It is not so much a matter of finding what you asked for, but of getting the permission to equip it on this ship.`
			`	"Once the ship is ready, it will be waiting for you on Darkcloak, since that is where I will make the necessary modifications."`
				decline
			label thief

> 			`	She seems upset, "I told you not to take any weapons, what were you thinking? That was not officially part of the duels so you can get away with it, but I certainly do not approve of it. Don't count on me to give you any special treatment for this ship.`

Reviewer: ```suggestion
			`	She seems upset, saying, "I told you not to take any weapons, what were you thinking? That was not officially part of the duel so you can get away with it, but I certainly do not approve of it. Don't count on me to give you any special treatment for this ship.`
```

		not "<uhai> 16: Strength Of A Leader: Si-Ra: done"
	on offer
		conversation
			branch "small fleet"
				"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 3 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 + "ships: Superheavy" * 18 <= 18
			`You find Si-Ra and He-Yi waiting for you on the spaceport. Si-Ra starts speaking to you, "You have brought too many ships, come back with only your flagship, or a few lighter ships, on another day, so we may duel!"`
				defer
			label "small fleet"
			`You find Si-Ra and He-Yi waiting for you on the spaceport. Si-Ra asks, "Are you ready for war, <last>?" It seems to be a rhetorical question, and he does not wait for you to answer at all. He adds, "All Unfettered are expected to be ready for war, at any time."`
			`	Si-Ra comes closer and takes you aside to speak to you, privately.`
			`	"You should know I do not approve of your rise of power, even though there is probably nothing I can do about it. Fight well, your life may depend on it."`
				to display
					"midas: wronged" > 1

> 					"unfettered rep: Si-Ra" <= -2

Reviewer: Is this an and or an or?

Author: default is and

				`	"On second thought, I'll come back later."`
					goto later
				`	"Now that it's here I'm starting to realize I don't want to be a part of your war. I'm out.`
					decline
			branch "mad Si-Ra"
				or
					"midas: wronged" > 1
					"unfettered rep: Si-Ra" < 0
			`	"And may you die ready as well. Let us fight, on the same terms of honor as before. Take off when you're ready, I will be waiting for you. Once you defeat me you will be able to fight the Tejolis."`
				accept
			label "mad Si-Ra"
			action
				set "unfettered: mad Si-Ra"
				event "unfettered: unbribable challengers"
			`	"And may you die ready as well. Let us fight! Take off when you're ready, I will be waiting for you. Once you defeat me you will be able to fight the Tejolis."`
				accept
			label later
			`	"Be sure to come back soon. War cannot wait forever."`
				defer
	on enter
		conversation
			branch "cheating bugger"
				"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 3 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 + "ships: Superheavy" * 18 > 18

> 			`You receive an anonymous hail with a bird insigna on it, "What did you do to get these people upset? They're not the kind you want to upset! Be warned, Si-Ra is not intending to play fair - meaning you don't have to either. I made sure he will not be getting any reinforcements, the rest is up to you, but do not let him disable your ship, and don't count on him accepting bribes either!"`

Reviewer: ```suggestion
			`You receive an anonymous hail with a bird insignia on it, "What did you do to get these people upset? They're not the kind you want to upset! Be warned, Si-Ra is not intending to play fair - meaning you don't have to either. I made sure he will not be getting any reinforcements, the rest is up to you, but do not let him disable your ship, and don't count on him accepting bribes either!"`
```

		to despawn
			has "unfettered: defeated Si-Ra"
		personality disables staying heroic target launching
		conversation
			`"You have defeated me, but He-Yi will not be as easy to beat! Land to repair, and then take off when you're be ready to face him. Don't forget, you are not to destroy this ship!" He launches an escape pod.`
			action
				set "unfettered: defeated Si-Ra"
	# if you made him be kicked from Midas he'll be angry and not respect the rules of the fight, in which case you may kill him
	# he'd get demoted for this anyway and runs away in an escape pod no matter what happens
	npc disable
		government "Hai (Unfettered Challenger)"
		ship "Si-Ra's angry Beetle" "Hellfire"
		to spawn
			has "unfettered: mad Si-Ra"
			"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 3 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 + "ships: Superheavy" * 18 <= 18
		to despawn
			has "unfettered: defeated Si-Ra"
		personality vindictive nemesis heroic target launching
		conversation

> 			`"Dammit! You just humiliated me, again? I take comfort in knowing He-Yi will most likely give you a lesson! Land to repair, and then take off when you're be ready to face him."`

Reviewer: ```suggestion
			`"Dammit! You just humiliated me, again? I take comfort in knowing He-Yi will most likely give you a lesson! Land to repair, and then take off when you're ready to face him."`
```

mission "<uhai> 16: Strength Of A Leader: He-Yi"
	landing
	deadline 2
	repeat
	source "Darkcloak"
	name `Strength of a Leader`
	description `Prove your strength against He-Yi in battle, and then land again on <destination> on the same day.`
	substitutions
		<question> "How does your ship cloak?"
			not "<uhai> 20: Bigeminate Sigil: Cloak: done"
		<question> "Why is your hull black?"
			has "<uhai> 20: Bigeminate Sigil: Cloak: done"
	to offer
		has "<uhai> 16: Strength Of A Leader: Si-Ra: done"
		not "<uhai> 16: Strength Of A Leader: He-Yi: done"
	on offer
		dialog `He-Yi comes to see you on your ship. "Are you ready to fight now? The rules are the same as always."`
	on enter
		conversation
			branch "cheating bugger"
				"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 3 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 + "ships: Superheavy" * 18 > 18
			`He-Yi hails you, "May crossing swords sharpen our blades<, unfettered title> <last>."`
				accept
			label "cheating bugger"

> 			`He-Yi hails you, "What is this big fleet you field? Are you scared? You should be, but coming with numbers only makes you appear weak! Consider this challenge [void, canceled], come back to me again when you are ready."`

Reviewer: ```suggestion
			`He-Yi hails you, "What is this big fleet you field? Are you scared? You should be, but coming with numbers only makes you appear weak! Consider this challenge [void, canceled], and come back to me again when you are ready."`
```

			`	"Most of us prefer not to use weapons of [aliens, untrue] because they think it means those are [more refined than, superior to] ours. I just recognize their [advantages, utility] in some scenarios."`
				goto questions
			label enough
			`	"I have answered enough of your questions. Some things should remain [a mystery, concealed]. Knowledge is power, after all."`
				to display
					"unfettered rep: He-Yi" < 3
			label clearconditions
			action
				clear "unfettered: questions"
				clear "unfettered: sword choice"
				clear "unfettered: hull choice"
				clear "unfettered: cloak choice"
				clear "unfettered: weapon choice"
			`	"The time will come soon, prepare yourself for [war, tragedy]: bring us Jump Drives, train in challenge battle, and when we are ready and only then, we will strike."`

event "joined the unfettered"
	government "Escort"
		swizzle 4
	"reputation: Wanderer" <?= -1

> 	set "global: unfettered human"

Reviewer: Global? Does this mean that any new pilot that is started is also an unfettered human?

Author: nah I just add this for a future achievement

				accept
	on waypoint
		dialog
			`It seems that the <npc> is in this system, and a Syndicate attack just launched!`
			`Judging by the marks on its hull, it will likely land to make repairs and restock ammo soon. You may be able to get the item without too much risk that way, and bribing them will mean secure landing. Or you could always just board it now and be done with it. You could even sabotage the ship to make your job easier, or take some more valuables with you.`
	# note: this only works because the fleets do not fight if you are not in the system.
	# If that was to change we'd need a temporary fleet with an event changing the reputation.
	npc
		personality coward plunders vindictive staying
		government "Mhairi's Legion"
		system "Alcyone"
		fleet "Large Northern Pirates" 3
		fleet "Small Northern Pirates" 5
	npc
		personality derelict uninterested coward
		government "Mhairi's Legion"
		system "Alcyone"
		fleet "Large Northern Pirates"
		fleet "Small Northern Pirates"
	npc
		personality heroic staying entering disables
		to despawn
			has "<uhai> 17: Spy On Hai B: offered"
		government "Syndicate"
		system "Alcyone"

> 		fleet "Large Syndicate" 3

Reviewer: The Syndicate will be annihilated here, so should this number be higher?

Author: they're meant to get rekt, I dont like fleet battles that randomly decide an outcome

Author: well mb I should bring it up a bit but then smbd would have to test it

Reviewer: No, you’re right. I’m just thinking from the Syndicate’s point of view, "We’ll send some of our big warships. Three to six is enough. They got annihilated?! Too bad, other things to do." I’m not sure quite makes sense. Of course, they’re all about profit, but…

				die
			label luckyish
			`	You get close up behind him and are about to pull your gun... when he draws a sudden breath and begins to turn around! Fortunately, you're quicker, and you have your firearm pointed at him before he can do anything. Obviously experienced with hostage situations, and prioritizing his own safety, he doesn't try to call for help. You present yourself to the group outside, with the gun still pointed at your hostage. The pirates look at each other and begin laughing before you die in a barrage of laser fire from the group, annihilating you and your hostage. It seems that they couldn't care less about their companions' lives.`
				die
			label steal
			`	You manage to enter the ship discreetly. Some people will always notice, but you try to pose as a spaceport worker there to assist with the repairs, using equipment you find lying around. You should be able to go pretty much anywhere on the ship. You don't have any idea which part of the ship is actually damaged though, so your cover could easily be blown. There is no going back now, except maybe if you choose to provoke the pirate warlord into a fight to get the item.`
			label search

> 			`	You won't be able to search the ship forevever, and it would be wise to think carefully about your next course of action.`

Reviewer 2: ```suggestion
			`	You won't be able to search the ship forever, and it would be wise to think carefully about your next course of action.`
```

		has "<uhai> 12: Outfitter Upgrade: done"
		not "<uhai> 18: Hai Language: active"
	on offer
		log "Minor People" "Mysterious Herald" "A Herald of He-Yi that does not want to reveal their true identity. They seem to have a lot of culture, and stay calm in most situations."
		conversation
			`You exit your ship and head to the spaceport as usual, but you cannot shake the sensation that you are being followed. Looking around you see a curious figure with a bird mask: the Herald. They head your way, aware of the fact they have been spotted.`
			`	As they get closer you feel like the color of the mask has slightly changed, but their voice seems the same "It did not take you long to spot me, did it? You have bested me<, unfettered title> <last>!" They say, bowing their head so little you barely notice it. "Rest assured, I am not just here to watch over you; my master has yet another task to ask of you before the war starts."`
			choice
				`	"I am listening."`
					goto task
				`	"Why were you watching me?"`
				`	"First tell me, why have you not introduced yourself yet?"`
					goto masked
				`	"Later."`
					defer
			label watching
			action
				set "uhai choice: watching"

> 			`	They look around discreetely - more so than earlier - before speaking in a lower tone, "Come, let us have a walk." You follow them, and they continue talking on the way, "Why does the To'or fight to the death? Is it pride? Is it protection of their loved ones, or a matter character? There are often multiple reasons for why things happen the way they do, and we do not always know them all."`

Reviewer 2: ```suggestion
			`	They look around discreetly - more so than earlier - before speaking in a lower tone, "Come, let us have a walk." You follow them, and they continue talking on the way, "Why does the To'or fight to the death? Is it pride? Is it protection of their loved ones, or a matter character? There are often multiple reasons for why things happen the way they do, and we do not always know them all."`
```

				goto choose
	on visit
		dialog "You have landed on <destination> but either the food or the Mystery Cube are still on one of your escorts in another system."
	on complete
		clear "uhai choice: cube in space"
		clear "uhai choice: keep cube"
		outfit "Mystery Cube" -1
		log `Placed a mysterious cube in a clearing on Hai-home. It should give intel on Hai ship movements and creation to the Unfettered.`
		payment 5000000
		dialog
			`You land, and the Hai start loading their food onto your ship. You tell them you want to stretch your legs (which isn't entirely a lie), and they don't appear to suspect a thing. You find a clearing and choose to bury the cube just below the ground. You remember to mark the exact spot, and then you return to your ship.`
			`You then notice that you have received a generous payment of <payment>. It appears they somehow learned of it.`

# 18 is optional, in side missions because you can learn Hai through the Hai diplomats
# idea: add a duel against the Herald where they have a Water Bug wearing an Octopulse
mission "<uhai> 19: Scout Wanderer Systems"

> 	name `Know your enemy`

Author: I dont know which book but I love the sound of that

> 	name `Know your enemy`

Author: oooooooooh yeah I read that one

				`	"Why were you watching me?"`
				`	"First tell me, why have you not introduced yourself yet?"`
					goto masked
				`	"Later."`
					defer
			label watching
			action
				set "uhai choice: watching"
			`	They look around discreetly - more so than earlier - before speaking in a lower tone, "Come, let us have a walk." You follow them, and they continue talking on the way, "Why does the To'or fight to the death? Is it pride? Is it protection of their loved ones, or a matter character? There are often multiple reasons for why things happen the way they do, and we do not always know them all."`
			`	They then look at you directly, "You should know that there are some among us that do not appreciate the status you have been given, or what happened with the Alphas."`
			choice
				`	"What was the task again?"`
					goto task
				`	"Can you tell me who I should be afraid of?"`
				`	"What is a To'or?"`
					to display
						not "unfettered: to'or knowledge"
					goto toor
			action
				set "unfettered: warned about scarred"
			`	"You should fear the Scarred. I will say no more, for information has a price."`
				goto task
			label toor
			action
				set "unfettered: to'or knowledge"

> 			`	"It is the most mighty creature we know of. They live on Hai-home, in small but resilient communities, where every member has earned their place, and where nobody is left behind. They never run away from a fight, and rightly so, since they are the most impressive creatures on the planet."`

Reviewer: Interesting to note that the Hai are less impressive.

Author: I imagine these as carapaced monsters that are strong so yeah def more impressive

				goto askmore
			label masked
			`	"When I wear that mask, I may not reveal my identity to anyone, that is my burden as a Herald of He-Yi. I do not wear this mask all the time, of course, only when I am on mission for him, or in his war room."`
			label askmore
			choice
				`	"Can you tell me more about the task?"`
				`	"Why were you watching me earlier?"`
					to display
						not "uhai choice: watching"
					goto watching
			label task
			`	"We want you to survey all systems belonging to our enemies, to assess if they have some backup fleets or potential allies nearby."`
			branch neutral
				"reputation: Wanderer" >= 0
			choice
				`	"But I am at war with them. It would be at a risk for me."`
			action
				"unfettered rep: Herald" --
				"unfettered rep: He-Yi" --

> 			`	They seem a bit angry, for once, "Again? In the last mission we had arranged for them to not shoot at you, what did you do?? Well there is nothing we can do for you this time, I just hope you have some means of hiding, or a very agile ship. Theirs are not that impressive but they do have long range missiles, meaning you should probably install some Chameleon Anti-Missiles on your ship."`

Reviewer: ```suggestion
			`	They seem a bit angry, for once. "Again? In the last mission we had arranged for them to not shoot at you, and what did you do? Well, there is nothing we can do for you this time. I just hope you have some means of hiding, or a very agile ship. Their spacecraft are not that impressive, but they do have long range missiles, meaning you should probably install some Chameleon Anti-Missiles on your ship."`
```

			label neutral
			choice
				`	"I'm in."`
					accept
				`	"Why can't you do it yourselves?"`
			`	"He-Yi is currently busy, too busy to go do a survey in his personal ship, and he will not let anyone else fly it, including me. That means we have no way to do the scouting ourselves without risking lives. Even his ship risks suffering malfunctions and be then cut off from us, behind enemy lines. We cannot risk that!"`
			choice
				`	"Got it, you need me. I'll do it then."`
					accept
				`	"I'm not interested in taking this kind of risk either."`
					decline
	waypoint "Chy'chra"
	waypoint "Pik'ro'iyak"
	waypoint "Ik'kara'ka"
	waypoint "Chirr'ay'akai"
	waypoint "Sich'ka'ara"
	waypoint "Kiro'ku"
	waypoint "Ka'pru"
	waypoint "Ap'arak"
	waypoint "Ka'ch'chrai"
	waypoint "Ek'kek'ru"
	waypoint "Si'yak'ku"
	waypoint "Iyech'yek"
	waypoint "Kiru'kichi"
	waypoint "Es'sprak'ai"
	waypoint "Aya'k'k"
	on waypoint

> 		dialog `You have scouted all of the Wanderer's systems. It's time to come back to <destination> for your reward.`

Reviewer: ```suggestion
		dialog `You have scouted all of the Wanderer systems. It's time to go back to <destination> for your reward.`
```

	on complete
		clear "uhai choice: watching"
		payment 1000000
		conversation

> 			`You find a message from the Herald on your terminal. "You should transmit the scans to the local authorities, we trust them to handle this kind of information. Please, accept these <payment> credits in advance.`

Reviewer: ```suggestion
			`You find a message from the Herald on your terminal. "You should transmit the scans to the local authorities, for we trust them to handle this kind of information. Please, accept these <payment> credits in advance.`
```

			`	You bring the scans to the authorities, as requested, and return to the spaceport.`

mission "<uhai> 19: Weird Aliens Spotted"
	landing
	invisible
	source
		attributes "unfettered"
	destination "Warfeed"
	substitutions
		"<pug aliens>" "weird aliens"
		"<pug aliens>" "Pug"
			has "main plot completed"
	to offer
		has "language: Hai"
		has "<uhai> 12: Outfitter Upgrade: done"
		not "<uhai> 18: Hai Language: active"
	waypoint "Pug Iyik"
	on waypoint
		dialog `These aliens do not look like the rest of the Wanderers at all! You should probably warn the Unfettered about it, on <destination>.`
	on complete
		conversation
			`Do you want to warn He-Yi's herald about the <pug aliens> north of Wanderer space?`
			choice
				`	(Speak to him about the weird aliens.)`
					to display
						not "main plot completed"
				`	(Speak to him about the Pug.)`
					to display
						has "main plot completed"
					goto pug
				`	(Do not warn the Unfettered about it.)`
					decline
			action
				payment 150000

> 				log `Warned the Unfettered about the weird aliens presence north of the Wanderers.`

Reviewer: ```suggestion
				log `Warned the Unfettered about the presence of the weird aliens north of the Wanderers.`
```

			`	"Please accept <payment> for that."`
			label "show arfecta?"
			choice
				`	(Warn him of the Arfecta.)`
					to display
						has "Pug Angry at Being Farmed: offered"
				`	(Show him the schematics of the Arfecta I have.)`
					to display
						has "ship model (all): Pug Arfecta"
					goto arfectadesigns
				`	(Leave.)`
					decline
			action
				set "unfettered: warned of arfecta"
				"unfettered rep: Herald" += 2
				payment 250000
			`	"Did you not say these Pug, as you call them, are masters of deception?`
				to display
					has "main plot completed"

> 			`	"We already know that they have somehow tricked the Wanderers into blindly worshiping them. That ship could have tricked your sensors. Nevertheless you deserve a reward for all of this intel, I shall double the payment."`

Reviewer: ```suggestion
			`	"We already know that they have somehow tricked the Wanderers into blindly worshipping them. That ship could have tricked your sensors. Nevertheless you deserve a reward for all of this intel, so I shall double the payment."`
```

			not "unfettered: Mera line"
		<welcomed> "welcomed"
		<welcomed> "[rewarded, welcomed]"
			not "unfettered: Mera line"
	to offer
		has "salary: Unfettered"
	on offer
		conversation
			`Not often do you see so much activity in Unfettered space. It looks like a major undertaking is about to happen, and you will probably be a part of it. When you land, Mera is already in the process of spreading a message for all Unfettered to hear, her voice somehow present across the entire spaceport.`
			`	She speaks in Hai but she made sure to provide you with the translation on the device she gave you back when you were preparing to battle the Auxiliary. You would easily be able to read it to get a full translation.`
				to display
					not "unfettered: Mera line"
			branch speak
				not "unfettered: Merora translate"
			`	Thankfully you spot Merora, the young Hai that helped you translate some time ago, back in the temple on this very same planet, when the elder's meeting with the public was happening.`
				to display
					has "unfettered: Merora translate"
			`	As always you have your translator with you, ready for action.`
				to display
					has "language: Hai"

> 			`	You also spot a translator or two near the stage, apparently for public use. They may be there for the occasional pirate the Unfettered may make use of, but there does not seem to be anyone using them here.`

Reviewer 2: Should this display if the line above displays?

			choice
				`	(Go speak with Merora.)`
					to display
						has "unfettered: Merora translate"
				`	(Grab the translator.)`
					to display
						not "language: Hai"
					goto speaktake
				`	(Use your translator.)`
					to display
						has "language: Hai"
					goto speak

> 			`	You notice she seems to have made a few friends since the last time you saw her, and she is so busy discussing with them that she hardly notices you until you are right in front of her. She stands there for a few seconds, maw open, before gaining back focus and smiling with joy. "<first>! It's been a while." She presents her comrades to you, "These are X and Y, they work on the same ship I do. To be frank we don't see battle often, but I've had my fair share of it already, so I'm more than happy with that!" She says the last part with a bit of shame, but honesty.`

Reviewer 2: > These are X and Y

:mad_everyone:

					to display
						has "language: Hai"
					goto speak

> 			`	You notice she seems to have made a few friends since the last time you saw her, and she is so busy discussing with them that she hardly notices you until you are right in front of her. She stands there for a few seconds, maw open, before gaining back focus and smiling with joy. "<first>! It's been a while." She presents her comrades to you, "This is Rick, a human that grew up in Hai space and then proved himself trough other means than battles." You shake hands and present yourself. "And this is Kenep, she left Hai space some time ago, to get inspiration for her art. They work on the same ship I do. To be frank we don't see battle often, but I've had my fair share of it already, so I'm more than happy with that!" She says the last part with a bit of shame, but honesty.`

Reviewer 2: ```suggestion
			`	You notice she seems to have made a few friends since the last time you saw her, and she is so busy discussing with them that she hardly notices you until you are right in front of her. She stands there for a few seconds, maw open, before gaining back focus and smiling with joy. "<first>! It's been a while." She presents her comrades to you, "This is Rick, a human that grew up in Hai space and then proved himself through other means than battles." You shake hands and present yourself. "And this is Kenep, she left Hai space some time ago, to get inspiration for her art. They work on the same ship I do. To be frank we don't see battle often, but I've had my fair share of it already, so I'm more than happy with that!" She says the last part with a bit of shame, but honesty.`
```

		log `Gathered materials for the Unfettered shipyards. They're building sizable ships here.`
		outfit "Lead" -50
		outfit "Silver" -20
		payment 500000
		conversation
			`You bring your cargo to the local Unfettered shipyards, and see huge ships being built in the distance. You also notice that not all materials are brought to the shipyards, some are simply sent to more rural areas. You may want to check the shipyards, later on.`
			`	One of the Unfettered team leaders comes to speak to you, "My greatest thanks for these materials, they will be crucial in repairing our houses, probably more so than for our ships. Please, if you have the time, try to bring us more materials, we post missions on the job board when we need them the most. In the meantime, take these <payment> as a token of my gratitude."`

mission "<uhai> 10: Titanium"
	name `Gather Titanium`
	description `Mine 40 tons of Titanium from <waypoints> for <payment>.`
	source "Firelode"
	waypoint "Ik'kara'ka"
	destination "Darkcloak"
	to offer
		has "salary: Unfettered"
		"reputation: Wanderer" >= 0
	on offer

> 		dialog "Do you want to go get Titanium from <waypoints> as Mera suggested?"

Reviewer 2: Why do you ask a question in a popup window, where you can't have choices?

Author: when a dialog is present in an on offer it acts as a decline/accept option

			label asteroidtuto
			action
				set "unfettered: asteroid tuto"

> 			`	She seems a bit surprised, "You've never mined asteroids? I recommend you install an asteroid scanner so your ship can target them automatically. Just try to keep your ship stable around one and shoot it as you would any other ship. Keep in mind they have no shields, and you might not want to waste missiles on them, since they don't shoot back. One thing everyone may not know is that once you've exhausted all the asteroids in a system, you may as well land or change system knowing they'll be back the next time you launch!"`

Reviewer 2: ```suggestion
			`	She seems a bit surprised, "You've never mined asteroids? I recommend you install an asteroid scanner so your ship can target them automatically. Just try to keep your ship stable around one and shoot it as you would any other ship. Keep in mind they have no shields, and you might not want to waste missiles on them, since they don't shoot back. If you ever run out of targets, you can fly around for a couple of days and come back; new asteroids will likely enter the system in that time."`
```

	description `Mine 40 tons of Titanium from <waypoints> for <payment>.`
	source "Firelode"
	waypoint "Ik'kara'ka"

> 	destination "Darkcloak"

Reviewer 2: Are you sure about this?

Author: yeah I want you to bring it directly to her on Firelode

				"Splinter"
				"Hauler"
				"Scrapper"
	npc kill
		government "Midas"
		personality disables plunders harvests staying
		system "Sumar"
		ship "Marauder Firebird (Bars)" "Dark Angel"
		to despawn
			has "outfit (all): Loot"
		conversation

> 			`You manage to take out the leader of the fleet, and you can feel them lose coherance. A captain from another ship hails you. "Get out of here, now. We will not pursue you."`

Reviewer 2: ```suggestion
			`You manage to take out the leader of the fleet, and you can feel them lose coherence. A captain from another ship hails you. "Get out of here, now. We will not pursue you."`
```

				`	"This seems too risky for me."`
					decline
	on waypoint

> 		dialog `It seems that the <npc> is in this system, and a syndicate attack just launched. Judging from the marks on its hull, it will likely land to make repairs and restock ammo soon. You may be able to get the item without too much risk that way, and bribing them will mean secure landing. Or you could always just board it now and be done with it. You could even sabotage the ship to make your job easier, or take some more valuables with you.`

Reviewer 2: Capitalize the Syndicate. Also, it's `Judging by`.

(I can't make suggestions for some reason.)

					"outfit (all): Ionic Turret Prototype" > "turrets owned"
			branch normal
				not "unfettered rep: Mera"
			`	"Since this was partly a test-run, what suggestions would you make to improve the ship, so I can make changes to its outfits, before I finally give this ship to you?"`
			label suggestions
			choice

> 				`	"The ship is fine, but I could use an extra outfit to modify the loadout myself."`

Reviewer 2: >"The ship is fine, but I could use an extra outfit to modify the loadout myself."

What does this mean? Do you want extra outfit space? One more of a specific outfit? One random outfit lying around?

> 				`	"Can you explain to me why you named the ship Hera?"`

Reviewer 2: It looks like this choice doesn't hide the option after it's taken? Not sure it's necessary, but I think the text flow from this option would work better that way.

			`	"Since this was partly a test-run, what suggestions would you make to improve the ship, so I can make changes to its outfits, before I finally give this ship to you?"`
			label suggestions
			choice

> 				`	"No need to modify the loadout, but I could use an extra outfit all the same."`

Reviewer 2: ~~This doesn't fix the issue.~~

				`	"Can you explain to me why you named the ship Hera?"`
					to display

> 						not "unfettered: Hera"

Reviewer 2: Do you clear this condition?

Author: on purpose no, I think the names of the ships matter enough to keep them

	on offer
		conversation
			`You are starting to get quite used to Unfettered spaceports: the dilapidation of different non-essential structures is beginning to feel normal. You feel like you need a change of scenery, so you wander to a side of the spaceport you have not explored before, but it turns out not to be very different.`

> 			`	Your thoughts are interrupted by a local Unfettered wearing a strange mask resembling some kind of a bird, who seems to have a very important message to give you given how fast he is speaking. "He-Yi has asked me to deliver you a message! It has come to his attention that an item of great importance to us is currently in the possession of an unfettered human warlord." You guess that he means a pirate leader. "The warlord can be found in <waypoints>. We want you to go and retrieve that item right now - by any means."`

Reviewer 2: ```suggestion
			`	Your thoughts are interrupted by a local Unfettered wearing a strange mask resembling some kind of bird, who seems to have a very important message to give you given how fast he is speaking. "He-Yi has asked me to deliver you a message! It has come to his attention that an item of great importance to us is currently in the possession of an unfettered human warlord." You guess that he means a pirate leader. "The warlord can be found in <waypoints>. We want you to go and retrieve that item right now - by any means."`
```

		government "Syndicate"
		system "Alcyone"
		fleet "Large Syndicate"
	# last so <npc> refers to this one
	npc board
		personality staying target
		to despawn
			has "<uhai> 17: Spy On Hai B: active"
		government "Mhairi's Legion"
		system "Alcyone"
		ship "Leviathan (Cube)" "Degenerate"
		on disable

> 			dialog `The Degenerate is disabled, but be careful not to destroy before you get the item or it may be lost.`

Reviewer 2: ```suggestion
			dialog `The Degenerate is disabled, but be careful not to destroy it before you get the item or it may be lost.`
```

	on offer
		log "Minor People" "Mysterious Herald" "A Herald of He-Yi that does not want to reveal their true identity. They seem to have a lot of culture, and stay calm in most situations."
		conversation
			`You exit your ship and head to the spaceport as usual, but you cannot shake the sensation that you are being followed. Looking around you see a curious figure with a bird mask: the Herald. They head your way, aware of the fact they spotted.`
			`	As they get closer you feel like the color of the mask has slightly changed, but their voice seems the same "It did not take you long to spot me, did it? You have bested me<, unfettered title> <last>!" They say, bowing their head so little you barely notice it. "Rest assured, I am not just here to watch over you; my master has yet another task to ask of you before the war starts."`
			choice
				`	"I am listening."`
					goto task
				`	"Why were you watching me?"`
				`	"First tell me, why have you not introduced yourself yet?"`
					goto masked
				`	"Later."`
					defer

> 			label watching

Author: oh right no but there should be haha

			`	They look around discreetely - more so than earlier - before speaking in a lower tone, "Come, let us have a walk." You follow them, and they continue talking on the way, "Why does the To-or fight to the death? Is it pride? Is it protection of their loved ones, or a matter character? There are often multiple reasons for why things happen the way they do, and we do not always know them all."`
			`	They then look at you directly, "You should know that there are some among us that do not appreciate the status you have been given, or what happened with the Alphas."`
			choice
				`	"What was the task again?"`
					goto task
				`	"Can you tell me more?"`
				`	"What is a To-or?"`
					goto toor
			action
				set "unfettered: warned about scarred"

> 			`	"Very well: Scarred. I will say no more, for information has a price."`

Reviewer 2: ~~What does this mean?~~
What is he telling you more about?

	waypoint "Iyech'yek"
	waypoint "Kiru'kichi"
	waypoint "Es'sprak'ai"
	waypoint "Aya'k'k"
	on waypoint
		dialog `You have scouted all of the Wanderer's systems. It's time to come back to <destination> for your reward.`
	on complete
		payment 1000000
		conversation
			`You find a message from the Herald on your terminal. "You should transmit the scans to the local authorities, we trust them to handle this kind of information. Please, accept these <payment> credits in advance.`

> 			`	"I think you deserve a small explanation as to why these aliens up north are hostile to us: as you probably already know, we are planning to invade them shortly. But, to be fair, we have already started doing a few skirmishes and raids, notably to assess their strength, but mostly for food. They have huge ships transporting a thousand tons of cargo each, that lack manoeuvrability and are therefore prime targets for our tracker pods."`

Reviewer 2: ```suggestion
			`	"I think you deserve a small explanation as to why these aliens up north are hostile to us: as you probably already know, we are planning to invade them shortly. But, to be fair, we have already started doing a few skirmishes and raids, notably to assess their strength, but mostly for food. They have huge ships transporting a thousand tons of cargo each that lack maneuverability and are therefore prime targets for our tracker pods."`
```
